php/commands/rewrite.php

```diff
@@ -306,6 +310,27 @@ function apache_get_modules() {
 			}
 		}
 	}
+
+	/**
+	 * Display a warning if --skip-plugins or --skip-themes are in use.
+	 *
+	 * Skipping the loading of plugins or themes can mean some rewrite rules
+	 * are unregistered, which may cause erroneous behavior.
+	 */
+	private static function check_skip_plugins_themes() {
+		$skipped = array();
+		if ( WP_CLI::get_config( 'skip-plugins' ) ) {
+			$skipped[] = 'plugins';
+		}
+		if ( WP_CLI::get_config( 'skip-themes' ) ) {
+			$skipped[] = 'themes';
+		}
+		if ( empty( $skipped ) ) {
+			return;
+		}
+		$skipped = implode( ' and ', $skipped );
+		WP_CLI::warning( sprintf( "Some rewrite rules may be missing because %s weren't loaded.", $skipped ) );
```

Comment:
Should be `weren\'t`. Double quotes introduce additional processing that single quotes don't.

But I have no strong opinion no this. Will approve and merge.